src/formats/palette.h

```diff
@@ -22,6 +22,28 @@
 extern "C" {
 #endif
 
+// All sprites have holes, these holes need to be transparent.
+#define SPRITE_TRANSPARENT_INDEX 0
```

Comment:
Instead of naming these per purpose, could we instead have reusable defined ?

#define DEFAULT_TRANSPARENCY 0
#define NO_TRANSPARENCY -1

and then the special, specific ones (90, D0, etc). That would potentially let us reuse the generic ones in other code, too.

Reply:
I can define DEFAULT_TRANSPARENCY and NO_TRANSPARENCY and set the defines that are currently 0 and -1 to the new defines. But would like to keep them separately and purposefully named when they are used in the c files.
This helps me to differentiate them, and it's easier to debug them when they are defined individually and are all stored in one place.

Reply:
Ok. As discussed on discord, let's do tranparency.h or somesuch.